### ISSUE TITLE
new resources and things for said resources 

### ISSUE THREAD

**Issue by ehwhynot** [Anuken/Mindustry-Suggestions#1850]
**Describe the content or mechanics you are proposing.**

the new resources are meteoric iron, iron, cobalt, lithium and magnetic alloy. 

starting with meteoric iron/ meteoric iron ore, it could be mined then smelted into 2 different resources. iron and cobalt, cobalt could then be used as fuel for turrets that destroy the spores (I don't know about that one but cobalt is toxic.) or cobalt could be used to make magnetic alloy (I'll get to that later.) 
iron could be used to make magnetic weapons or defenses, or as ammo. 
The magnetic weapons would be something like a long range gun that fires a bullet that pierces due to its speeds, but it would work better with less inaccuracy, or the weapon could use magnets to drag in enemy air units and keep them stationary, like the parallax but it would use less power, not deal damage and have a stronger pull. it would be used to pull air units together so their easier for other turrets to hit. 
the magnetic defenses would be somewhat like a forcefield but instead any bullet that is metal based would be destroyed or collected and enemy air units would either be destroyed or disorientated. 

lithium would be gotten through a extractor that extracts it from water or if you think that's too easy it could instead be extracted from a mineable resource called pegmatite or pegmatitic mineral. lithium would be used for powerful batteries, flame ammo/ weapons or used for nuclear reactions, possibly for power or as a weapon. 
the batteries would be build with lithium, graphite and copper, you can have other things I just think these are necessary resources for the battery. 
as ammo I think it would be a more powerful version of pyratite, or could be used in a flamethrower type weapon, which would have range similar to the arc, and be able to be fueled by pyratite, lithium or maybe slag. 
lithium is somewhat unstable so you could use it in nuclear missiles or possibly nuclear reactions for a power source. 
the nuclear reactor would require at least magnetic alloy, graphite and iron to build, and probably other resources, you would have to feed it lithium and cryofluid. 

magnetic alloy would require cobalt, lithium, graphite and thorium, possibly iron but I feel that would make it to easy for magnetic alloy schematics because you wouldn't need to have somewhere for the iron to go. 
magnetic alloy could be used to build magnetic weapons and maybe even be a ammo but I feel it would have to be pretty damaging if it were a weapon. you could also use magnetic alloy to build the magnetic weapon that drags in enemy air units, or even a magnetic wall. I don't know about the properties of the wall but maybe if its damaged and a enemy dies near it it repairs itself. 

most of this is based of some facts I found while researching. 

**Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**

1. - [X] I have done a quick search in the list of suggestions to make sure this has not been suggested yet.
2. - [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
3. - [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
4. - [X] I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by SteelBlue8** [comment]
This largely sounds like the territory of a mod rather than something for the base game, I have to say. Certainly interesting features, but not necessarily very vanilla.

**Comment by Kyllingene** [comment]
Here is a lesson I learned from Extra Credits (**TL;DR at bottom**): Depth ≠ complexity ≠ quality. Let me explain.

Complexity = How many things (items, units, blocks, etc) you have available
Depth = How much you can do with those things
Quality (not always) ≈ Your depth to complexity ratio

When you add complexity, you do several things. First of all, you make it harder (sometimes, *much* harder) for someone to start playing. Imagine you started playing Minecraft, and immediately there are 7 different passive mobs,
13 different types of wood, and 5 types of starting ore. That would be a mess to sort out in your head, and you might quit.

Second of all, you make it take longer to achieve goals. When you have to go through one factory to get a resource, then use that resource to make another, then build a block with that that lets you get another, then use that resource to *finally* make the factory that gives you access to the block you desire, it gets confusing really quickly. Again, a mess to sort out in your head.

However, when you add *depth*, you make the game harder to learn, yes, but not in memory work. Instead of starting up the game and immediately having to memorize 20 different blocks, you start up and work with what you have. The challenge is then in achieving goals with what you are given, instead of the challenge being to figure out what block does what.

Now, many games benefit from complexity. Take games like Factorio, for example. They are built around complex systems with tons of different blocks, and lots of people love that. It is not inherently bad. However, I do not believe that Mindustry is supposed to be like that. I think it is supposed to be minimalist.

#### TL;DR: Possibly a little too much complexity for Mindustry, which is supposed to be somewhat minimalist. However, not an inherently bad idea.
